worldengine/draw.py

```diff
@@ -693,7 +696,7 @@ def draw_scatter_plot(world, size, target):
         for x in range(world.width):
             if world.is_land((x, y)):
                 t = world.temperature_at((x, y))
-                p = world.humidity['data'][y, x]
+                p = world.humidity_at((x, y))
```

Comment:
This is something I keep thinking about. Both notations show up at various places and while the latter is a littler prettier, is there an actual chance of this becoming important? There are millions over millions of function-calls triggered by the "pretty kind of notation" ( and it's O(n²) ); especially calling things like `is_land()` contribute a non-insignificant amount of time to the overall generation time. I'm not sure if that is truly worth it.